docs-new/docs/graphql/core/api-reference/index.mdx

```diff
@@ -0,0 +1,111 @@
+---
+description: Hasura API reference
+keywords:
+  - hasura
+  - docs
+  - API
+  - API reference
+slug: index
+---
+
+# API Reference
+
+## Available APIs
+
+| API                            | Endpoint                                         | Access           |
+|--------------------------------|--------------------------------------------------|------------------|
+| GraphQL                        | [/v1/graphql](#graphql-api)                      | Permission rules |
+| Relay                          | [/v1beta1/relay](#relay-api)                     | Permission rules |
+| Legacy GraphQL                 | [/v1alpha1/graphql](#graphql-api)                | Permission rules |
+| Schema *(> v2.0)*              | [/v2/query](#schema-api)                         | Admin only       |
+| Metadata *(> v2.0)*            | [/v1/metadata](#metadata-api)                    | Admin only       |
+| Schema/Metadata *(deprecated)* | [/v1/query](#schema-metadata-api-deprecated)     | Admin only       |
+| Restified GQL                  | [/api/rest](#restified-graphql-api)              | GQL REST Routes  |
+| Version                        | [/v1/version](#version-api)                      | Public           |
+| Health                         | [/healthz](#health-check-api)                    | Public           |
+| PG Dump                        | [/v1alpha1/pg_dump](#pg-dump-api)                | Admin only       |
+| Config                         | [/v1alpha1/config](#config-api)                  | Admin only       |
+| Explain                        | [/v1/graphql/explain](#explain-api)              | Admin only       |
+
+### GraphQL API {#graphql-api}
+
+All GraphQL requests for queries, subscriptions and mutations are made to the GraphQL API.
+
+See details at [GraphQL API Reference](/graphql/core/api-reference/graphql-api/index.mdx).
+
+### Relay API {#relay-api}
+
+Hasura exposes a Relay schema for GraphQL requests for queries, subscriptions and mutations.
+
+See docs at [Postgres: Relay schema](/graphql/core/databases/postgres/schema/relay-schema.mdx).
+
+See details at [Relay GraphQL API Reference](/graphql/core/api-reference/relay-graphql-api/index.mdx).
+
+### Schema API {#schema-api}
+
+Hasura exposes a schema API for directly executing SQL on the underlying Postgres.
+
+This is primarily intended to be used as an `admin` API to manage the Hasura schema.
+
+See details at [Schema API Reference](/graphql/core/api-reference/schema-api/index.mdx).
+
+### Metadata API {#metadata-api}
+
+Hasura exposes a metadata API for managing metadata.
+
+This is primarily intended to be used as an `admin` API to manage the Hasura metadata.
+
+See details at [Metadata API Reference](/graphql/core/api-reference/metadata-api/index.mdx).
+
+### Schema / metadata API (Deprecated) {#schema-metadata-api-deprecated}
```

Comment:
The original file seems to not have `-deprecated` suffixed in id. So, lets remove it here too

`#schema-metadata-api` should suffice

Reply:
Fixed